kuma/settings/prod.py

```diff
@@ -0,0 +1,25 @@
+from .common import *  # noqa
+
+ATTACHMENT_HOST = 'mdn.mozillademos.org'
+
+DATABASES['default']['OPTIONS'].update({
+    'charset': 'utf8',
+    'use_unicode': True,
+})
```

Comment:
That's some oddity between the various settings, the prod settings have it, but the local ones don't. I'm afraid of upgrading the charset to utf8 and unicode locally to not break the database somehow.